components/desktop/mate/mate-system-monitor/patches/03-default-icon.patch

```diff
@@ -0,0 +1,14 @@
+--- mate-system-monitor-1.20.1/src/iconthemewrapper.cpp.orig	2018-10-16 01:12:23.890830040 +0000
++++ mate-system-monitor-1.20.1/src/iconthemewrapper.cpp	2018-10-16 01:12:30.260851246 +0000
+@@ -12,7 +12,10 @@
+ {
+     try
+     {
+-      return Gtk::IconTheme::get_default()->load_icon(icon_name, size, flags);
++	  if (Gtk::IconTheme::get_default()->has_icon(icon_name))
++	    return Gtk::IconTheme::get_default()->load_icon(icon_name, size, flags);
++      else
++	    return Gtk::IconTheme::get_default()->load_icon("image-missing", size, flags);
```

Comment:
Some lines have tabs & spaces, some do only spaces, after patching the code is not aligned correctly.